.github/CODEOWNERS

```diff
@@ -346,9 +346,9 @@ packages/react-components/react-calendar-compat/stories @microsoft/cxe-prg
 packages/react-components/react-infolabel @microsoft/cxe-prg
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32888&commitId=11706ae3c4f8781d4c69b5f02fe621d64183a015&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|4| Changed|
|Avatar Converged.badgeMask.chromium.png|41| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1166| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2798| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>